app/templates/administrative-units/administrative-unit/ministers/new.hbs

```diff
@@ -0,0 +1,359 @@
+{{page-title "Nieuw"}}
+{{breadcrumb "Nieuw"}}
+
+<div class="">
+  <div class="au-u-margin au-u-margin-bottom-large">
```

Comment:
Is missing a class here? Include the scroll because the table does not show completely.

Reply:
And also, after selecting a person, I can't see the page completely.

Reply:
ahhh I should test things on a smaller screen as well :smile: